--- src/browsers/playlist3-playlist-editor.old	2013-03-25 23:11:04.126241397 +0100
+++ src/browsers/playlist3-playlist-editor.c	2013-03-25 23:15:56.804582414 +0100
@@ -566,20 +566,24 @@
 }
 static void playlist_editor_clear_playlist(GtkWidget * item, gpointer data)
 {
-    GtkWidget *delete;
-	gchar *path = g_object_get_data(G_OBJECT(item), "path");
-    gchar *message = g_strdup_printf(_("Are you sure you want to clear the playlist: '%s'"), path);
-
-    delete = gtk_button_new_from_stock(GTK_STOCK_CLEAR);
-
-    g_object_set_data_full(G_OBJECT(delete), "path", g_strdup(path), g_free);
-    g_signal_connect(G_OBJECT(delete), "clicked", 
-            G_CALLBACK(playlist_editor_clear_playlist_real), NULL);
-
-    /* show message */
-    playlist3_message_show(pl3_messages, message,USER_FEEDBACK); 
-    playlist3_message_add_widget(pl3_messages, delete);
-    g_free(message);
+
+
+  playlist_editor_clear_playlist_real(item, data);
+
+    /*  GtkWidget *delete; */
+    /* 	gchar *path = g_object_get_data(G_OBJECT(item), "path"); */
+    /* gchar *message = g_strdup_printf(_("Are you sure you want to clear the playlist: '%s'"), path); */
+
+    /* delete = gtk_button_new_from_stock(GTK_STOCK_CLEAR); */
+
+    /* g_object_set_data_full(G_OBJECT(delete), "path", g_strdup(path), g_free); */
+    /* g_signal_connect(G_OBJECT(delete), "clicked",  */
+    /*         G_CALLBACK(playlist_editor_clear_playlist_real), NULL); */
+
+    /* /\* show message *\/ */
+    /* playlist3_message_show(pl3_messages, message,USER_FEEDBACK);  */
+    /* playlist3_message_add_widget(pl3_messages, delete); */
+    /* g_free(message); */
 }
 
 static void playlist_editor_load_playlist(GtkWidget * item, gpointer data)
